qa/workunits/rbd/rbd_mirror.sh

```diff
@@ -411,31 +497,39 @@ done
 testlog "TEST: disable mirror while daemon is stopped"
 stop_mirrors ${CLUSTER1}
 stop_mirrors ${CLUSTER2}
-set_pool_mirror_mode ${CLUSTER2} ${POOL} 'image'
+if [ "${RBD_MIRROR_MODE}" = "${JOURNAL_MODE}" ]; then
+  set_pool_mirror_mode ${CLUSTER2} ${POOL} 'image'
+fi
 disable_mirror ${CLUSTER2} ${POOL} ${image}
 if [ -z "${RBD_MIRROR_USE_RBD_MIRROR}" ]; then
   test_image_present ${CLUSTER1} ${POOL} ${image} 'present'
 fi
 start_mirrors ${CLUSTER1}
 wait_for_image_present ${CLUSTER1} ${POOL} ${image} 'deleted'
-set_pool_mirror_mode ${CLUSTER2} ${POOL} 'pool'
-enable_journaling ${CLUSTER2} ${POOL} ${image}
+if [ "${RBD_MIRROR_MODE}" = "${JOURNAL_MODE}" ]; then
```

Comment:
Let's test for snapshot-based mirroring here and make its branch first, like it is everywhere else.